pages/landing/hipaa/template/pricing/index.html

```diff
@@ -0,0 +1,50 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Pricing
+excerpt: Configure the plan just right for your team.
+permalink: /hipaa-cloud-compliance/pricing/
+
+extra_nav_links:
+  - name: Gruntwork HIPAA
+    url: /hipaa-cloud-compliance
+
+pricing:
+  - title: Gruntwork HIPAA Early Access
+    description: Early access subscribers will receive a CIS staging environment when invited<br>
+      and a fully HIPAA-compliant production infrastructure when it's ready.
+    price: 3060
+    strike_price: 4080
```

Comment:
I think HIPAA compliance is likely closer to $10K/mo full price. The discounted price we can experiment with.

Reply:
Bumped monthly price to 10K in 52d308f2